airflow/api_connexion/openapi/v1.yaml

```diff
@@ -4302,7 +4302,7 @@ components:
             If set, don't actually run this operation. The response will contain the task instance
             planned to be affected, but won't be modified in any way.
           type: boolean
-          default: false
+          default: true
```

Comment:
This should be false

Reply:
Per the conversation with @pierrejeambrun and @hussein-awala, I changed it to false